docs/usage/examples/opentelemetry.md

```diff
@@ -86,7 +118,7 @@ Start setup using this command inside the folder containing the files created in
 docker-compose up
 ```
 
-This command will start an [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector-contrib) and an instance of [Jaeger](https://www.jaegertracing.io/).
+This command will start an [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector-contrib), an instance of [Jaeger for traces](https://www.jaegertracing.io/) as well Prometh.
```

Comment:
```suggestion
This command will start:

- an [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector-contrib)
- an instance of [Jaeger for traces](https://www.jaegertracing.io/)
- and Prometheus
```

Let's put the items in a list.

Please check me on the last line! I think you meant `Prometheus` instead of `Prometh`?